packages/components/src/Tooltip/src/Tooltip.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+    useColorSchemeContext,
+    useStyledSystem,
+    type StyledComponentProps,
+    type StyledSystemProps
+} from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, useContext, type ForwardedRef } from "react";
+import { composeRenderProps, Tooltip as RACTooltip, useContextProps, type TooltipProps as RACTooltipProps } from "react-aria-components";
+
+import { HopperProvider } from "../../HopperProvider/index.ts";
+import { TextContext } from "../../typography/index.ts";
+import { composeClassnameRenderProps, cssModule, ensureTextWrapper, SlotProvider, type BaseComponentDOMProps } from "../../utils/index.ts";
+
+import { TooltipContext } from "./TooltipContext.ts";
+import { TooltipTriggerContext } from "./TooltipTriggerContext.ts";
+
+
+import styles from "./Tooltip.module.css";
+
+export const GlobalTooltipCssSelector = "hop-Tooltip";
+
+type PropsToOmit = "triggerRef" | "UNSTABLE_portalContainer" | "placement" | "containerPadding" | "offset" | "crossOffset" |
```

Comment:
containerPadding feels like a prop we do not need, since we allow containerProps. What do you think?

Reply:
hmm I guess I can see your point. It should maybe be set to 0 (default is 12) and then we can add our own margin on the tooltip container. This would only work right if we never add an arrow though.

Reply:
actually this would not work.. The tooltip ends up too far away, I forgot about this:
![image](https://github.com/user-attachments/assets/44e49e37-f40a-4c9a-b779-8cc26cccad3b)